Sources/Controllers/2.0/Settings/ImportExport/OAPluginInstalledViewController.mm

```diff
@@ -303,21 +360,30 @@ - (UIView *) getHeaderForTableView:(UITableView *)tableView withFirstSectionText
 
 - (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section
 {
-    if (section == 1)
+    if ([_data[section][0][@"type"] isEqualToString:kCellTypeMap])
```

Comment:
Error-prone code. Please use variables for sections and assign them to section indexes if they exist, otherwise leave them at -1.